secp-ffm/src/test/java/org/bitcoinj/secp/ffm/EcdhTest.java

```diff
@@ -0,0 +1,7 @@
+package org.bitcoinj.secp.ffm;
+
+/**
+ *
+ */
+public class EcdhTest {
+}
```

Comment:
I generally do not like empty class protos. It gives a false sense (in this case of test coverage).

Reply:
This will be removed in a squash.